_projects/2023/100516517/100516517.Rmd

```diff
@@ -0,0 +1,798 @@
+---
+title: "Final Project - Candela Gómez Blanco"
+description: |
+  A short description of the post.
+categories: "2023"
+author: Candela Gómez Blanco
+date: "`r Sys.Date()`"
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true
+---
+
+# Introduction
```

Comment:
The main title is already level 1. Headings insisde the post should start at level 2 (`##`).